target_athena/utils.py

```diff
@@ -57,140 +55,16 @@ def flatten_record(d, parent_key=[], sep="__"):
-    field_definitions = []
-    new_level = level + 1
-    indentation = new_level * tab
-    for name, attributes in schema.items():
-        cleaned_name = "`{}`".format(name)  # if name.lower() in keywords else name
-        if attributes["type"] == "object":
-            field_definitions.append(
-                "{indentation}{name}{separator}STRUCT<\n{definitions}\n{indentation}>".format(
-                    indentation=indentation,
-                    name=cleaned_name,
-                    separator=type_separator,
-                    definitions=generate_field_definitions(
-                        attributes["properties"], new_level
-                    ),
-                )
-            )
-        elif attributes["type"] == "array":
-            extra_indentation = (new_level + 1) * tab
-            if attributes["items"]["type"] == "object":
-                closing_bracket = "\n" + indentation + ">"
-                array_type = "STRUCT<\n{definitions}\n{indentation}>".format(
-                    indentation=extra_indentation,
-                    definitions=generate_field_definitions(
-                        attributes["items"]["properties"], new_level + 1
-                    ),
-                )
-            else:
-                closing_bracket = ">"
-                array_type = attributes["items"]["type"].upper()
-            field_definitions.append(
-                "{indentation}{name}{separator}ARRAY<{definitions}{closing_bracket}".format(
-                    indentation=indentation,
-                    name=cleaned_name,
-                    separator=type_separator,
-                    definitions=array_type,
-                    closing_bracket=closing_bracket,
-                )
-            )
-        elif isinstance(attributes["type"], list):
-            types = [_ for _ in attributes["type"] if _ != "null"]
-            field_definitions.append(
-                "{indentation}{name}{separator}{type}".format(
-                    indentation=indentation,
-                    name=cleaned_name,
-                    separator=type_separator,
-                    type="STRING"
-                    # type=types[0].upper()
-                )
-            )
-        else:
-            field_definitions.append(
-                "{indentation}{name}{separator}{type}".format(
-                    indentation=indentation,
-                    name=cleaned_name,
-                    separator=type_separator,
-                    # type=attributes['type'].upper()
-                    type="STRING",
-                )
-            )
-    return field_separator.join(field_definitions)
-
 
-def generate_create_database_ddl(
-    database: str="default"
-)-> None:
-    return f"CREATE DATABASE IF NOT EXISTS {database};"
+    key = "{prefix}{stream_name}/{timestamp}.{object_format}".format(
+        prefix=prefix, 
+        stream_name=stream_name, 
+        timestamp=timestamp, 
+        object_format=object_format)
```

Comment:
With f-strings, this can be simplified further....

```suggestion
    key = f"{prefix}{stream_name}/{timestamp}.{object_format}"
```

Reply:
Yeah, there's like at least 3 different string interpolation methods being used in this code base 😁